OLCNE/scripts/provision.sh

```diff
@@ -611,10 +721,11 @@ main () {
   passwordless_ssh
   msg "Oracle Linux base software installation complete."
   # All nodes are up, orchestrate installation
-  if [[ -n "${OPERATOR}" ]]; then
+  if [[ ${OPERATOR} == 1 ]]; then
     certificates
     bootstrap_olcne
     deploy_kubernetes
+    deploy_modules
     fixups
```

Comment:
I would run the fixups before deploying anything else, mainly for `flannel`...

Reply:
I'm not sure it matters and I wanted all the modules to go down before we did anything Vagrant/VirtualBox specific (wherever possible). But I'm happy to be convinced otherwise.

Reply:
As OLCNE currently do not support multi-nics the Vagrant deployment requires some hacks to ensure the cluster uses `eth1`.  
After installing the kubernetes OLCNE module the `flannel` DaemonSet is on the _wrong_ interface and needs to be re-deployed. This is done in the `fixups` function and the cluster is not fully functional before this is done.  
We should not deploy anything on the K8s cluster before it is fully operational.